components/compliance-service/ingest/pipeline/publisher/compliance.go

```diff
@@ -116,18 +121,8 @@ func insertInspecProfile(msg message.Compliance, profile *relaxting.ESInspecProf
 			"profile_id": profile.Sha256,
 		}).Debug("Ingesting inspec_profile")
 		start := time.Now()
-		profileExists, err := client.ProfileExists(profile.Sha256)
```

Comment:
We used to check for profile existence for each profile coming in the report. Reports can contain one or multiple profiles. The logic has been changed and we only query once elasticsearch asking for all the profiles in the report at once with the `ProfilesMissing` method